vendor/lief/src/logging.hpp

```diff
@@ -91,15 +91,19 @@ class Logger {
 
   template <typename... Args>
   static void err(const char *fmt, const Args &... args) {
+    const char *prefix = "Error: ";
```

Comment:
What do you think about adding the prefix to all the other kinds of logs as well, i.e., trace, debug and info?

Reply:
I think if you patch https://github.com/nodejs/postject/blob/master/vendor/lief/src/logging.cpp#L55 to include the `%l` flag it should already do what we want?

Reply:
Nice catch @robertgzr 🙌🏼